md2conf/converter.py

```diff
@@ -190,6 +201,11 @@ def transform(self, child: ET.Element) -> Optional[ET.Element]:
         if child.tag == "p" and len(child) == 1 and child[0].tag == "img":
             return self._transform_image(child[0])
 
+        # <p>[[_TOC_]]</p>
+        # <p>[TOC]</p>
+        elif child.tag == "p" and "".join(child.itertext()).startswith(("[[TOC]]", "[TOC]")):
```

Comment:
Thanks for the update, I hope you agree that this looks much nicer. ;)

Documentation for GitLab says:
> Add either the `[[_TOC_]]` or `[TOC]` tag on its own line

Both of your code comments (`<p>[TOC]</p>` and `<p>[[_TOC_]]</p>`) seem to confirm. This means we shouldn't need the call to `startswith`, do we? We could use string equality. I would like us to be as strict as possible to avoid picking up something that looks like a table of contents but in fact is not (e.g. someone writing TOC in the body of text for *terms of contract*).